### PR TITLE
Ovms windows mediapipe build enabling

### PR DESCRIPTION
**Pull request by rasapala**
### 🛠 Summary

JIRA CVS-154715
Enabling mediapipe on windows builds aka MEDIAPIPE_DISABLE=0
- startup --output_user_root=C:/b_tmp for long paths fix.
- matching MP glog repo
- mediapipe_windows_http config got linux-mediapipe-http build path.
- disabling some calculators for windows
- fixing compilation for google::gflags on windows - FLAGS_minloglevel

### 🧪 Checklist

- [ ] Unit tests added.
- [ ] The documentation updated.
- [x] Change follows security best practices.
``

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -179,10 +179,14 @@ http_archive(
...
 
-git_repository(
+# 2020-08-21
+http_archive(
     name = "com_github_glog_glog",
-    remote = "https://github.com/google/glog",
-    tag = "v0.5.0",
+    strip_prefix = "glog-3a0d4d22c5ae0b9a2216988411cfa6bf860cc372",
+    sha256 = "170d08f80210b82d95563f4723a15095eff1aad1863000e8eeb569c96a98fefb",
+    urls = [
```
Comment (atobiszei) [inline WORKSPACE:187]: Please add info what is the sha/tag/branch of this package

**Review by dkalinowski — APPROVED** [review]

File: common_settings.bzl [hunk 2]
```diff
@@ -68,6 +68,13 @@ def create_config_settings():
         name = "not_fuzzer_build",
         negate = ":fuzzer_build",
     )
+
+    selects.config_setting_group(
+        name = "mediapipe_windows_http",
```
Comment (atobiszei) [inline common_settings.bzl:73]: rename to sth more meaningfull, like is_windows_or_mediapipe_is_disabled

File: third_party/mediapipe_calculators/BUILD [hunk 3]
```diff
@@ -105,13 +102,20 @@ cc_library(
...
         "@mediapipe//mediapipe/calculators/video:motion_analysis_calculator",
         "@mediapipe//mediapipe/calculators/video:flow_packager_calculator",
         "@mediapipe//mediapipe/calculators/video:box_tracker_calculator",
         "@mediapipe//mediapipe/calculators/video:box_detector_calculator",
         "@mediapipe//mediapipe/calculators/video:tracked_detection_manager_calculator",
         "@mediapipe//mediapipe/calculators/video:video_pre_stream_calculator",
-    ],
+    ]  + select({
+        "//conditions:default": [
+            "@mediapipe//mediapipe/calculators/core:packet_cloner_calculator", # TODO windows: stdc++20 required
```
Comment (atobiszei) [inline third_party/mediapipe_calculators/BUILD:113]: You probably need to adjust whitelist of calculators and adjust for Win/Lin

Comment (rasapala) [reply third_party/mediapipe_calculators/BUILD:113]: This will be done when enabling tests.

**Review by atobiszei — APPROVED** [review]

**Pull request merged**